public/js/redux/chartsRedux.js

```diff
@@ -28,9 +30,15 @@ const chartsRedux = State({
         }
     },
 
-    updateComposerVsIncopy(state, composerVsInCopyData) {
+    updateComposerVsIncopy(state, { chartData, startDate, endDate }) {
+        const { composerResponse, inCopyResponse } = chartData;
+        const range = endDate.diff(startDate, 'days');
+        const composerData = composerResponse.data.length < range ?  fillMissingDates(startDate, endDate, composerResponse.data).sort(compareDates) : composerResponse.data;
```

Comment:
I'd agree that long term the API response should perhaps be more generic. Initially it seemed right to have a very specific API response but this is no longer the case so it's currently sitting between specific and generic. But I think returning no data from the API when there were no articles started in a specific system and handling this client side makes sense.